gk/fib.c

```diff
@@ -1136,7 +1136,7 @@ del_fib_entry_locked(struct ip_prefix *ip_prefix, struct gk_config *gk_conf)
  * @ip_prefix the IP prefix,
```

Comment:
I can't comment on the line `@gateway the gateway address informaiton.` above, but could you fix the typo `informaiton`?